lib/katello/content/file.rb

```diff
@@ -0,0 +1,4 @@
+Katello::ContentPluginManager.register(::Katello::Repository::FILE_TYPE) do
+  description "File Repository types"
+  allow_creation_by_user false
```

Comment:
we've never allowed creation of 'file' type repos by the user.  They are only currently used for the ISO repos.  

I guess we could open it up if that would simplify things.